plugins/catalog-backend/migrations/20220116144621_remove_legacy.js

```diff
@@ -14,6 +14,15 @@
  * limitations under the License.
  */
 
-export { CatalogBuilder } from './CatalogBuilder';
-export { createRouter } from './router';
-export type { RouterOptions } from './router';
+// @ts-check
+
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function up(knex) {
+  await knex.schema.dropTable('entities_relations');
```

Comment:
Have you tested this migration with Postgres? I seem to remember there was some issues dropping some of these tables before, but maybe it's fine?

Reply:
Yeah I tried to be extra cautious because of that exact reason. What I did was:

- Checkout master
- Revert back to the shape of `packages/backend/src/plugins/catalog.ts` before the next engine was invented (`new CatalogBuilder`, using the `higherOrderComponents` etc)
- Start up the backend
- Log into postgres, validate that all of these three legacy tables are filled with data. Also check that the catalog API returns valid data
- Stop the backend
- Check out the PR branch, make sure that `catalog.ts` is now in the new style instead
- Start up the backend
- Validate that there are no warnings logs, that the legacy tables are gone, and that the new refresh loops work and API calls return valid data from the new tables